.github/workflows/backport.yml

```diff
@@ -5,12 +5,16 @@ on:
       - closed
       - labeled
 
+permissions:
+  pull-requests: write
+  contents: write
+
 jobs:
   backport:
     runs-on: ubuntu-latest
     name: Backport
     steps:
       - name: Backport
-        uses: tibdex/backport@v1.1.1
+        uses: tibdex/backport@v2
```

Comment:
It needs a `if` statement for `v2`.
https://github.com/tibdex/backport/blob/main/.github/workflows/backport.yml

Reply:
Good spot, thanks @climba03003, added in https://github.com/fastify/fastify/pull/3828/commits/f95458f735ce9ba532264a54eb8eebd5bb5a5f2b